packages/vscode-extension/src/migration/migrationTool/ts/mappings.json

```diff
@@ -0,0 +1,605 @@
+{
```

Comment:
Any required change in webpack?

Reply:
No, since we do not introduce any resource file.